CONTRIBUTING.md

```diff
@@ -82,21 +83,20 @@ You will find the Træfik executable in the `~/go/src/github.com/containous/trae
 
 If you happen to update the provider templates (in `/templates`), you need to run `go generate` to update the `autogen` package.
 
-### Setting up `glide` and `glide-vc` for dependency management
+### Setting up dependency management
 
-- Glide is not required for building; however, it is necessary to modify dependencies (i.e., add, update, or remove third-party packages)
-- Glide can be installed either via homebrew: `$ brew install glide` or via the official glide script: `$ curl https://glide.sh/get | sh`
-- The glide plugin `glide-vc` must be installed from source: `go get github.com/sgotti/glide-vc`
+[Dep](https://github.com/golang/dep) is not required for building; however, it is necessary to modify dependencies (i.e., add, update, or remove third-party packages)
 
-If you want to add a dependency, use `$ glide get` to have glide put it into the vendor folder and update the glide manifest/lock files (`glide.yaml` and `glide.lock`, respectively). A following `glide-vc` run should be triggered to trim down the size of the vendor folder. The final result must be committed into VCS.
+If you want to add a dependency, use `dep ensure -add` to have [Dep](https://github.com/golang/dep) put it into the vendor folder and update the [Dep](https://github.com/golang/dep) manifest/lock files (`Gopkg.toml` and `Gopkg.lock`, respectively).
 
-Care must be taken to choose the right arguments to `glide` when dealing with dependencies, or otherwise risk ending up with a broken build. For that reason, the helper script `script/glide.sh` encapsulates the gory details and conveniently calls `glide-vc` as well. Call it without parameters for basic usage instructions.
+A following `make prune-dep` run should be triggered to trim down the size of the vendor folder.
```

Comment:
This will not be necessary anymore when the next release of `dep` is out

Reply:
yes and the next release will come... https://github.com/golang/dep/pull/1219#issuecomment-355536070